apps/meteor/app/api/server/v1/users.ts

```diff
@@ -1255,6 +1256,25 @@ API.v1.addRoute(
 	},
 );
 
+API.v1.addRoute(
+	'users.getNames',
+	{ authRequired: true, validateParams: isUsersGetNamesParamsGETProps },
+	{
+		async get() {
+			const users = await Users.findByUsernames(this.queryParams.usernames, {
```

Comment:
We're not covering the case of `usernames` not being sent as a param here. It could be `undefined` when `userIds` is provided instead.

Reply:
@matheusbsilva137 isn't that handled by ajv )isUsersGetNamesParamsGETProps) ?

Reply:
Nice catch, I completely overlooked that part 🙈